en/components/autocomplete.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Autocomplete Directive
+_description: The igxAutocomplete directive provides a way to enhance a text input by showing a panel of suggested options provided by the developer.
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Autocomplete components, Angular Autocomplete directives, Angular Autocomplete controls
+---
+
+## Autocomplete
+<p class="highlight">
+
+The [**igxAutocomplete**]({environment:angularApiUrl}/classes/igxautocompletedirective.html) directive provides a way to enhance a text input by showing a [drop-down]({environment:angularApiUrl}/classes/igxdropdowncomponent.html) with suggested options that are provided by the developer.
+
+The [igx-autocomplete]({environment:angularApiUrl}/classes/igxautocompletedirective.html) suggestions will show once you start typing in the text input. By default, the first item is always highlighted, so that it can be easily selected using the `Enter` key. An other way of selecting an item is to click on it from the list, then the input value will be automatically updated. Also, the drop-down will close. When an item is selected, the `onItemSelected` event is fired, if this event is canceled, the item will not be selected and the drop-down will not close.
+The `igxAutocomplete` directive allows you to navigate through items and at the same time to have focus always on the input. The latter will allow you to continue typing in the input. You can use the `ESC` key to close the drop-down list.
+
+The [igx-autocomplete]({environment:angularApiUrl}/classes/igxautocompletedirective.html) uses the [igx-drop-down]({environment:angularApiUrl}/classes/igxdropdowncomponent.html) as a provider for the available options, which means that the available capabilities of the drop-down component can be used, these include grouping, templates and disabling of items and groups.
```

Comment:
remove "that" -> all the available.... can be used out of the box - including goruping...